.github/workflows/build-push.yml

```diff
@@ -0,0 +1,40 @@
+name: Build + Push to Azure
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Source branch to deploy"
+        required: false
+        default: "main"
```

Comment:
I think source branch should be required, and not default to main. This way someone doesn't accidentally publish our docs from the main branch, since we want our published docs to always be from a release branch.